build-utils/sentry-instrumentation.ts

```diff
@@ -132,21 +130,20 @@ class SentryInstrumentation {
     if (!this.Sentry) {
       return;
     }
-    if (this.hasInitializedBuild) {
-      this.transaction = this.Sentry.startTransaction({
-        op: 'webpack-build',
-        name: !this.hasInitializedBuild ? 'initial-build' : 'incremental-build',
-        description: 'webpack build times',
-        startTimestamp: startTime,
-        trimEnd: true,
-      });
-    }
 
-    this.transaction
+    const span = !this.hasInitializedBuild
```

Comment:
Refactored this a bit - this was actually not typed, we had duplicate `name` and `description` (which are really aliases for the same thing), so I aligned this to just a single `name`.